flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,281 @@
+                    entry = self.generate_tv_entry(parent_request)
+                    log.debug('Valid entry %s', entry)
+                    self._items.append(entry)
+                else:
+                    for child_request in parent_request["childRequests"]:
+                        for season in child_request["seasonRequests"]:
+                            # Seasons do not have approvals or available flags so include them all
+                            if self.config.get('type') == 'seasons':
+                                entry = self.generate_tv_entry(parent_request, child_request, season)
+                                if entry:
+                                    log.debug('Entry %s', entry)
+                                    self._items.append(entry)
+                            else:
+                                for episode in season['episodes']:
+                                    # check that the request is approved unless user has selected to include everything
+                                    if self.config.get('only_approved') and not episode.get('approved') or episode.get('approved'):
+                                        # Always include items that are not available and only include available items if user has selected to do so
+                                        if (self.config.get('include_available') and episode.get('available')) or not episode.get('available'):
+                                            entry = self.generate_tv_entry(parent_request, child_request, season, episode)
+                                            log.debug('Valid entry %s', entry)
+                                            self._items.append(entry)
+        return self._items
+
+    def add_movie(self, entry):
+        log.verbose('Adding entry %s', entry.get('title'))
+        auth_header = self.ombi_auth()
+
+        parsedurl = urlparse(self.config.get('base_url'))
+        url = '%s://%s:%s%s/api/v1/Request/movie' % (parsedurl.scheme, parsedurl.netloc, self.config.get('port'), parsedurl.path)
+        data = {'theMovieDbId': entry.get('tmdb_id')}
+        try:
+            return requests.post(url, json=data, headers=auth_header).json()
+        except RequestException as e:
+            raise plugin.PluginError('Unable to connect to Ombi at %s. Error: %s' % (url, e))
+
+    def add(self, entry):
+        if self.config.get('type') == 'movies':
+            if entry.get('tmdb_id'):
+                result = self.add_movie(entry)
+                if result.get('message'):
+                    log.verbose('%s', result.get('message'))
+                elif result.get('errorMessage'):
+                    log.verbose('%s', result.get('errorMessage'))
+                else:
+                    log.verbose('Unknown result')
+            else:
+                log.verbose('Skipping %s ombi requires an tmdb_id to add movies, try using the tmdb_lookup plugin', entry.get('title'))
```

Comment:
If it's a requirement, we should help the user out and use the tmdb_lookup plugin for them. 

```
from flexget.plugin import get_plugin_by_name

get_plugin_by_name('tmdb_lookup').instance.lookup(entry)
```